Makefile

```diff
@@ -50,16 +47,17 @@ nsxiv: $(OBJS)
 	@echo "CC $@"
 	$(CC) $(CFLAGS) $(CPPFLAGS) -c -o $@ $<
 
-$(OBJS): Makefile nsxiv.h commands.lst config.h config.mk
+$(OBJS): Makefile nsxiv.h commands.lst config.h lib_autodetect.mk
 options.o: version.h
 window.o: icon/data.h
 
-config.mk:
+lib_autodetect.mk:
```

Comment:
I think `autodetect.mk` is a better name just in case in the feature we want to autodetect inotify